x-pack/plugins/secops/public/components/load_more_table/index.tsx

```diff
@@ -209,9 +229,21 @@ export const BasicTableContainer = styled.div`
   flex-direction: row;
   flex-wrap: wrap;
   height: auto;
+  position: relative;
 `;
 
 const FooterAction = styled.div`
   margin-top: 0.5rem;
   width: 100%;
 `;
+
+const BackgroundRefetch = styled.div`
+  background-color: ${props => props.theme.eui.euiColorLightShade};
+  margin: -5px;
+  height: calc(100% + 10px);
+  opacity: 0.7;
+  width: calc(100% + 10px);
+  position: absolute;
+  z-index: 3;
+  border-radius: 5px;
+`;
```

Comment:
Great additions to the `load_more_table`! It's nice to not have all the tables changing size every time they reload data. We may want to discuss showing this background overlay when clicking `Load More` as well since you can still interact with the IP dropdown and the Uni/Bidirectional toggle in the middle of the load.